Code/GraphMol/SLNParse/SLNParse.h

```diff
@@ -45,14 +45,14 @@ namespace RDKit{
     void finalizeQueryMol(ROMol *mol,bool mergeHs);
   }
 
-  RWMol *SLNToMol(std::string smi,bool sanitize=true,int debugParse=0);
+  RWMol *SLNToMol(const std::string &smi,bool sanitize=true,int debugParse=0);
```

Comment:
Actually, the smiles parsing is subtly different as well, as opposed to trimming white space, the parser now reads up to the first white space.

So now "CCC foo" will parse.  I think this is fine, but  should be noted/reviewed.  It can easily be updated if necessary.

----
Brian Kelley

> On Sep 25, 2015, at 5:50 PM, Greg Landrum <notifications@github.com> wrote:
> 
> In Code/GraphMol/SLNParse/SLNParse.h:
> 
> > @@ -45,14 +45,14 @@ namespace RDKit{
> >      void finalizeQueryMol(ROMol *mol,bool mergeHs);
> >    }
> >  
> > -  RWMol *SLNToMol(std::string smi,bool sanitize=true,int debugParse=0);
> > +  RWMol *SLNToMol(const std::string &smi,bool sanitize=true,int debugParse=0);
> IGNORE THIS, it's just marking how far I read before having to leave.
> 
> —
> Reply to this email directly or view it on GitHub.
>